package.json

```diff
@@ -2,8 +2,8 @@
   "name": "electron-packager",
   "version": "17.1.2",
   "description": "Customize and package your Electron app with OS-specific bundles (.app, .exe, etc.) via JS or CLI",
-  "main": "src/index.js",
-  "types": "src/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/types.d.ts",
   "bin": {
     "electron-packager": "bin/electron-packager.js"
```

Comment:
issue: I'm pretty sure you'll need to update this file. (Although I don't know if I'd necessarily convert it to TypeScript, since it's basically a Node.js-version-checking shim for `src/cli`.)

Reply:
We're still shipping the `bin` directory to npm, so the path here is fine — I did forgot to update a `require` in that file from `src` to `dist` though 😅

That being said, do we really need this file? Like, if the user has e.g. Node 10 and we require Node 16, the package manager should prevent `packager` from being installed (unless they're using `--ignore-engines`, but then the user should be aware that things can break)

Reply:
You'd be surprised how many issues were filed back in the day about why there were SyntaxErrors even though the minimum engine has been specified for a very long time.

Reply:
I happened to find a relatively new instance of this with Electron Forge in the [Discord server](https://discord.com/channels/745037351163527189/1175850453637009518): the user was using Node.js 12 when Forge requires Node.js 16, and Forge definitely has engine constraints in its various package.json files.

Reply:
Yeah, now I see why the validation is there, we shouldn't trust `engines` at all. Adding a proper validation to the ~40 packages in the Forge monorepo is gonna be fun 😟

Reply:
I would just add it to the two CLI-facing packages: `create-electron-app` and `@electron-forge/cli`.